### DISCUSSION TITLE
Messages aren't sent

### DISCUSSION THREAD

**Discussion by Buckmode** [Extended-Thunder/send-later#438] [Q&A]
Forgive the length of this but I am trying to include as much info as possible at the beginning.

I am using Send Later vers. 9.0.10 with Thunderbird vers. 91.7.0 on a PC with Win 7 Pro.  I can create a message, set the Send Later, but the message does not go when it is supposed to.  In the Send Later window, I have the option to send in 5 minutes, which I chose.  (The "long" button at the bottom which I have to click says  end in 8 minutes.)  When the message finally does move, it just goes to my Outbox.  If I click on File, the option Send Unsent Messages is grayed out.  I tried clicking on the Send Now button from within Send Later but the pop up message says **Clicking “Send Now” sends the message immediately. It DOES NOT schedule the message to be sent at a specific time. For that, click the “Send around” button in the green box after entering a valid send time,**  However there is no Send Around button and there is no green box.  Can someone help me with this?  Thanks!

**Comment by jph-sendlater** [comment]
Whoa, that popup message is way outdated! Thanks for catching that.

On the other issues, it sounds like you're doing everything right, but when the "Send Unsent Messages" button is greyed out you have problems with your Thunderbird that are preventing the Send Later extension from doing its thing. It might be a full temporary directory in Windows. See here for details: https://extended-thunder.github.io/send-later/#full-temporary-directory-on-windows
